### ISSUE TITLE
[0.9.0] how to change weather

### ISSUE THREAD

**Issue by hdy117** [carla-simulator/carla#741]
Hello, thank you for your brilliant work. Now I have a question
In previous version, we can change weather by editting the .ini file.
In 0.9.0 version, how do I change weather?
I also find a dynamic weather class, can I change weather while simulation is running?
If yes, how?

**Comment by hdy117** [comment]
@nsubiron I think I have found the issue in Dynamic Weather. At line of 75 in file CarlaSettings.h, 

/** Load weather description from config files. (There may be overrides for each map). */
  UFUNCTION(BlueprintCallable, Category="CARLA Settings")
  void LoadWeatherDescriptions();

is defined. but maybe it is not used. so, WeatherDescriptions is empty.  

at line of  295

check(WeatherDescriptions.Num() > 0);

would throw an exception.

I called LoadWeatherDescriptions in the blueprint. 
Now it is working, I can change weather dynamically.
